src/app/course-list/course/component.ts

```diff
@@ -5,25 +5,27 @@ export class Course {
   id: number;
   name: string;
   num: string;
-  departmentCode: string;
-  minCredits:number;
-  maxCredits:number;
+  department_code: string;
+  department_id: number;
+  min_credits:number;
+  max_credits:number;
   description: string;
   sections: Section[];
 }
 
 @Component({
   selector: 'course',
   templateUrl: './component.html',
+  styleUrls: ['./component.scss']
 })
 export class CourseComponent {
   @Input() course: Course;
 
   /* A getter function for the range of credits based on the min and max.
    * When {{creditRange}} is used in the template, this function will be called. */
   public get creditRange() {
```

Comment:
Can we move these helper methods into the course / section classes? They should really be member functions. I know this was already set up like this but since we are making changes here I think it makes sense to include that in this PR.